docs/config/options.md

```diff
@@ -34,11 +50,17 @@ The casing option defines the casing of the functions generated by Zap.
 
 ### Example
 
-<CodeBlock code="opt casing = PascalCase" />
+<CodeBlock code="opt casing = camelCase" />
+
+## `write_checks`
 
-## Writechecks
+This option determines if Zap should check types when writing data to the network. This is useful for development and debugging, but can see some performance hits, and should be disabled in production.
 
-The writechecks option determines if conditional constraints should be valided on the just the receiver (`false`) or on both the sender and receiver (`true`).
+::: danger
+Zap only checks types that cannot be statically checked by Luau or TypeScript.
+
+For example, Zap will not check if a `string (20)` is a string, but it will check that the string is 20 characters long.
```

Comment:
As @nezuo suggested - we should explain how buffers don't require us to do all checks runtime

Reply:
This is a good idea, but I'm not sure where to put it in the docs